### ISSUE TITLE
Update request: nextcloud 27.0.0 → 27.0.1

### ISSUE THREAD

**Issue by AkechiShiro** [NixOS/nixpkgs#244506]
- Package name: nextcloud{27,26,25}
- Latest released version: 27.0.1

- Current version on the unstable channel: 27.0.0
- Current version on the stable/release channel: 27.0.0

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**
@Ma27 @schneefux @bachp @globin 

Other releases were made for major version 25 and 26 : 
- 
![image](https://github.com/NixOS/nixpkgs/assets/14914796/91347db1-48b6-4f67-b760-8242f3666b80)
On unstable : 
![image](https://github.com/NixOS/nixpkgs/assets/14914796/ebba2204-3782-4a3d-b40f-6723ac41be1b)
On 23.05 release channel : 
![image](https://github.com/NixOS/nixpkgs/assets/14914796/0b669743-dd24-4455-8d7d-71ac4d4e0807)

-----

Note for maintainers: Please tag this issue in your PR.

**Comment by Ma27** [comment]
> I'd like to work on a GitHub Actions that automatically, bumps the version if it is just a minor version bump, is that something possible ?

While I *think* we have GH Actions for a _very small_ subset of packages, I'm not a big fan of multiple infrastructures for upgrading packages considering that we also have @r-ryantm 

Also, when merging a Nextcloud update it's always a good idea for $maintainer to actually deploy this to a live instance for testing, so manual intervention is needed anyways and I don't consider the version bump that much of an additional burden %)

**Comment by AkechiShiro** [comment]
@Ma27 I agree with you, major version bumps should be thoroughly tested by someone with manual intervention, however for minor version such as all bumps in 27.y.x, I don't think this is a very big risk for something to break, if there are some integration test running before merging is done.

Also, this GitHub Actions/or using any bot could have the sole purpose of only notifying maintainers of a new version available.
I don't know if it is possible using @r-ryantm as I'm totally not familiar with the automatic workflows that are run in Nixpkgs.

For instance, an automatic issue to bump the version be created if and only if the issue doesn't exist and the version inside the master branch of Nixpkgs is older than the new release.

**Comment by RaitoBezarius** [comment]
@AkechiShiro https://nixos.org/manual/nixpkgs/stable/#special-variables `passthru.updateScript`

**Comment by SuperSandro2000** [comment]
Also FYI you don't need to paste screenshots for outdated versions, we believe you without them

**Comment by AkechiShiro** [comment]
@SuperSandro2000  No worries for the screenshots, I'll put text next time, I was just a bit lazy not really concerned about whether you'd believe or not

**Comment by RaitoBezarius** [comment]
Next time, consider sending a PR, it will go faster ;).
